package.json

```diff
@@ -8,7 +8,8 @@
     "dist"
   ],
   "scripts": {
-    "build": "node ./scripts/bundle.js && rollup -c"
+    "build": "node ./scripts/bundle.js && rollup -c",
+    "prettier": "prettier --write ."
```

Comment:
This should also be enforced in the build process, just not with `--write` but with `--check`. See what we have in Prada for comparison.

Reply:
Added `husky` pre-commit task.